crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -490,15 +477,25 @@ fn declare(
     );
     let tx = DeclareTransaction::new(
         starknet_api::transaction::DeclareTransaction::V2(declare_tx),
+        // FIXME incorrectly calculating hashes may be the case of the problems
+        TransactionHash(StarkFelt::from(1_u32)),
```

Comment:
Maybe lets just use `TransactionHash::default()` for now. It'll be easier to replace all occurrences of it in the future.

Reply:
And add a corresponding TODO here

Reply:
done